ztp/kube-compare-reference/optional/storage/StorageLVMSubscriptionNS.yaml

```diff
@@ -9,7 +9,4 @@ metadata:
         "workload.openshift.io/allowed: \"management\""
         "openshift.io/cluster-monitoring: \"true\""
```

Comment:
We can also get rid of this `requiredLabels` template helper...  In fact, it may be easier to just revert 8ec2e880 entirely.